10-angular/custom_directives/readme.md

```diff
@@ -97,41 +87,43 @@ So: let's make one!
 
 ## Grumble `show`
 
-- In `index.html`, add this custom directive: `<my-custom-directive></my-custom-directive>`. Refresh the page, and you shouldn't see anything.
+Make sure the local api is running at localhost:3000 - https://github.com/ga-dc/grumblr_rails_api
+
+- In `js/grumbles/index.html`, add this custom directive: `<my-custom-directive></my-custom-directive>`. Refresh the page, and you shouldn't see anything.
```

Comment:
Don't we still see all the grumbles?

"Refresh the page, and you shouldn't see any changes.  The new directive doesn't do anything... yet."